src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -1805,19 +1919,19 @@ METHOD(kernel_ipsec_t, add_system_enterprise, status_t,
 
 	struct system_enterprise_map_enterprise enterprise = {
 		.term_vlan_id = term_vlan_id,
+		.neigh_sub_interface = neigh_sub_interface,
```

Comment:
Keep initialization order to be the order the fields in the struct are defined. Specifically move `neigh_sub_interface` after the comment about `term_private_addr`